### ISSUE TITLE
Form save error if field handle is too long

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#499]
**Description**
After adding a new field with a very long label, the auto-generated handle caused the following error:
`Next yii\db\Exception: SQLSTATE[42000]: Syntax error or access violation: 1059 Identifier name 'craft_fmc_50_pleaseconfirmthatyouhavetherighttoaddaverylonglabel' is too long`

The save of the form is stopped because of this error (the loading spinner never ends, because of a JS error which is caused by this PHP error).
It would be useful to display an error after save if the field handle is too long and/or automatically shorten it if necessary.

**Steps to reproduce**
1. Create a new field with a very long label (example: _Please confirm that you have the right to add a very long label._)
2. Save the form

**Additional info**
- Plugin version: 1.4.7
- Craft version: 3.6.17
- Multi-site: No

**Comment by engram-design** [comment]
Strange, thought I took care of this ages ago https://github.com/verbb/formie/issues/135 but will take another look.

Was the field in a nested field (group or repeater)?

**Comment by romainpoirier** [comment]
I'm sorry, I'm not sure, I don't remember. This was done during a stress-test of a form, that I finally deleted / refactored.

**Comment by engram-design** [comment]
No problem, I'll investigate further!

**Comment by engram-design** [comment]
Looks like this was a group/repeater field, and only an issue when the project has a db prefix defined. Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.8](https://github.com/verbb/formie/releases/tag/1.4.8)
